routes/courseRoutes.js

```diff
@@ -54,6 +55,21 @@ router.post("/courses", async (req, res) => {
   }
 });
 
+router.get("/course/getHome", async (req, res) => {
+
+  res.send(["Hello course","Hello course 242424","Hello test"]);
+});
+
+//Route that fetched all subscribed courses of logged in user. It is using a fixed user now, but the out commented function is for the logged in user 
+router.get("/course/getSubscribedCourses", async (req, res) => {
```

Comment:
This URI doesn't match the standard we are following:
- The request type is spelled out
- It contains capitalization (separate with dashes instead)

Proposed change: `/course/subscribed-courses`